grouper/models.py

```diff
@@ -472,6 +474,9 @@ class Group(Model):
     canjoin = Column(Enum(*GROUP_JOIN_CHOICES), default="canask")
     enabled = Column(Boolean, default=True, nullable=False)
 
+    audit_id = Column(Integer, ForeignKey("audits.id"), nullable=True)
```

Comment:
By distinguishing "[global] audits" and "[group] reviews", one way to resolve the circular dependency would be to have groups point at audits, reviews point at groups and audits, and a relationship with backref between reviews and audits.